quick-start-with-htap.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Quick start with HTAP
+summary: Learn how to quickly get started with the TiDB HTAP.
+---
+
+# Quick Start Guide for TiDB HTAP
+
+This guide walks you through the quickest way to get started with TiDB's one-stop Hybrid Transactional and Analytical Processing (HTAP).
+
+> **Note:**
+>
+> The steps provided in this guide is ONLY for quick start, NOT for production. To explore more features of HTAP, see [explore HTAP](/explore-htap.md).
```

Comment:
Where does one fine the steps for production? Maybe explain why this isn't suitable for production etc

Reply:
I agree with Daniel's point. It's much better to say "for production deployments XYZ is recommended" than "this is not for production".

Reply:
```suggestion
> The steps provided in this guide are ONLY for quick start. For production deployments, [Explore HTAP](/explore-htap.md) is recommended. 
```